docs/source/pywren.scripts.rst

```diff
@@ -0,0 +1,38 @@
+pywren\.scripts package
```

Comment:
Good point. I added the generated files to the `gitignore`. The doc `rst` files are generated, but some of the webpages are done by hand.